iree_tests/onnx/import_tests.py

```diff
@@ -140,32 +200,26 @@ def import_onnx_files(test_dir_path, imported_dir_path):
     for i in range(len(test_inputs)):
         test_input = test_inputs[i]
         t = convert_io_proto(test_input, model.graph.input[i].type)
+        ty = get_io_proto_type(model.graph.input[i].type)
         if t is None:
             return False
         input_path = (imported_dir_path / test_input.stem).with_suffix(".npy")
         np.save(input_path, t)  # Only for ref, actual comparison with .bin
-        ss = get_shape_string(t)
```

Comment:
If this is no longer used, please remove it from https://github.com/nod-ai/SHARK-TestSuite/blob/main/iree_tests/onnx/import_tests_utils.py 

You could put `convert_proto_etype` and `get_io_proto_type` in that file too, so this file is smaller... or merge the two files together. I don't have a strong preference.

Reply:
Done.